docker-compose.link.yaml

```diff
@@ -6,7 +6,10 @@ version: '3'
 services:
   ffc-demo-web:
     networks:
+      - default

```

Comment:
This doesn't seem to be present on other repos, should it be added to others too?

Reply:
Yes, but only when they apply the other network changes in this PR.

Instead of specifying a custom network name in `docker-compose.yaml`, we're now letting Docker Compose create its default internal network for each stack. Services are automatically added to that network unless they specify other networks in the yaml config, as is the case here.

The default network's name is prefixed with the project name, so in this case it will show up in `docker network ls` as `ffc-demo-web_default`.